fireflower/core.py

```diff
@@ -28,6 +28,7 @@ class FireflowerStateManager(object):
     session = None
     structlog_threadlocal = False
     sentry = None
+    dogstatsd = None
```

Comment:
Name `datadog_dogstatsd` so it's more consistent with the method below (more clarity and less room for ambiguity for the reader).